spec/test_spec.rb

```diff
@@ -0,0 +1,63 @@
+require_relative '../book'
+require_relative '../person'
+require_relative '../student'
+require_relative '../teacher'
+require_relative '../class_room'
+require_relative '../rental'
+
+describe Book do
+  book = Book.new('Hadis', 'Bahlu')
+  it `test add book` do
+    expect(book.title).to include 'Hadis'
+  end
+  it `test title ` do
+    expect(book.author).to include 'Bahlu'
+  end
+end
```

Comment:
- [ ] it would be better to organize the test by putting each class in a separate file in order the tests will be clean and pro